Add lab 2-6 to BrendVandenEynde.md

BrendVandenEynde opened the pull request.

BrendVandenEynde committed:
feat: add labo 1

BrendVandenEynde committed:
Merge branch 'iamgoodbytes:main' into main

BrendVandenEynde committed:
Merge branch 'iamgoodbytes:main' into main

BrendVandenEynde committed:
Update README.md with project links

BrendVandenEynde committed:
add: markdown method to lab 5

BrendVandenEynde committed:
Update project layout in BrendVandenEynde.md

BrendVandenEynde committed:
fix: layout in BrendVandenEynde.md

BrendVandenEynde committed:
fix 2: layout

BrendVandenEynde committed:
fix 3: Layout

EviVermeeren commented:
Lab 1: voeg nog even de live link toe, was een samenwerking dus je mag die van je partner indienen. 
Lab 2: ziet er goed uit. Eilanden worden ingeladen, je kan ze opslaan, de pagina herladen en opnieuw doen verschijnen. 👍🏻
Lab 3: werkt, weer op basis van mijn locatie en een random pokémon via een API. 👍🏻
Lab 4: live link toevoegen, op basis van wat er in de opdracht staat. 

"Controles van die labo doe ik via https://f4rdq2.csb.app/ of  https://codepen.io/goodbytes/full/ywWQVb — gebruik die tool zelf om je routes te testen (fork deze code om je eigen oplossing in te sturen met je eigen API URL ingevuld)"

Lab 5: werkt, zotte database wel. Berichtje zou eigenlijk wel bovenaan moeten komen. 🧐👍🏻

"Make sure to use a reactive() array of messages in order to update them right away after submitting your new message (no page reload!)"

Lab 6: liefst nog even een GIF toevoegen ervan. Ook heb ik een link naar één pagina en kan ik dus niet testen of het werkt omdat ik het invoerveld niet heb... Nog even aanpassen! Het lijkt mij ook niet meteen live te updaten, daarom misschien best de gif.

EviVermeeren commented:
Aanvulling bij labo 6: 

![image](https://github.com/iamgoodbytes/dev5-portfolios-2023/assets/90692264/12f3bb16-a6cc-411b-b2f3-9e82c609ddbe)

BrendVandenEynde committed:
docs: Add Codesandbox link to BrendVandenEynde.md Description:
- Inserted a link in BrendVandenEynde.md that directs to the Codesandbox project
- Provides easy access to the online version for quick reference

iamgoodbytes commented:
lab1 git: ok, maar convential commit messages mankeren in lab 2, ...
lab2: ok
lab3: ok
lab4: geen werkende demo online
lab5: waarom geen deployment via vercel, werking ok
lab6: geen live updates (geen sockets), dit kan in je project nog
